### PR TITLE
kata-check: require kvm modules for amd64

### PR DESCRIPTION
**Pull request by marcov**
KVM modules are required when using QEMU or firecracker.

Fixes: #1985

Signed-off-by: Marco Vedovati <mvedovati@suse.com>

### CODE REVIEW

File: cli/kata-check_amd64.go [hunk 1]
```diff
@@ -116,21 +116,24 @@ func setCPUtype(hypervisorType vc.HypervisorType) error {
...
+					required: true,
 				},
 				kernelModvhostnet: {
-					desc: msgKernelVirtioNet,
+					desc:     msgKernelVirtioNet,
+					required: true,
 				},
 				kernelModvhostvsock: {
-					desc:     msgKernelVirtioVhostVsock,
-					required: false,
```
Comment (devimc) [inline cli/kata-check_amd64.go]: missing `required: true,` ?

Comment (marcov) [reply cli/kata-check_amd64.go]: vosck is not required, and by default required is false, so I just removed the line as it's redundant.

Comment (devimc) [reply cli/kata-check_amd64.go]: I didn't know that by default it's `false`, can we keep it explicit?

Comment (marcov) [reply cli/kata-check_amd64.go]: That's how [Go works](https://tour.golang.org/basics/12), but I am happy to set it explicitly to `false` if this gives more clarity.

Comment (devimc) [reply cli/kata-check_amd64.go]: thanks @marcov

**Comment by marcov** [comment]
All tests are good, but we need a pair of reviews :-)

Calling in the usual folks :grin: @jodh-intel @grahamwhaley @devimc @jcvenegas

**Pull request merged**